GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -87,6 +87,8 @@ public sealed partial class FormBrowse : GitModuleForm, IBrowseRepo
         private readonly TranslationString _undoLastCommitText = new TranslationString("You will still be able to find all the commit's changes in the staging area\n\nDo you want to continue?");
         private readonly TranslationString _undoLastCommitCaption = new TranslationString("Undo last commit");
 
+        private readonly TranslationString _comEmuNotFound = new TranslationString("Failed to found ConEmu.exe. Please reinstall Git Extensions.");
```

Comment:
As discussed, we don't expect this error to occur for an installed app. The issue is only expected to occur in dev time, so directing a user to reinstall would be misleading.

I would say something along the line of:
`ConEmu appears to be missing. Please perform a full rebuild and try again.`